flathub.json

```
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```

```suggestion
  "only-arches": ["x86_64"]
```

This is more inline with the intent